functional-tests/src/test/java/com/redhat/che/functional/tests/BayesianPackageJson.java

```diff
@@ -1,48 +0,0 @@
-/*
- * Copyright (c) 2016-2018 Red Hat, Inc.
- * This program and the accompanying materials are made
- * available under the terms of the Eclipse Public License 2.0
- * which is available at https://www.eclipse.org/legal/epl-2.0/
- *
- * SPDX-License-Identifier: EPL-2.0
- *
- * Contributors:
- *   Red Hat, Inc. - initial API and implementation
- */
-package com.redhat.che.functional.tests;
-
-import com.redhat.che.selenium.core.workspace.RhCheWorkspaceTemplate;
-import org.eclipse.che.selenium.core.workspace.InjectTestWorkspace;
-import org.eclipse.che.selenium.core.workspace.TestWorkspace;
-import org.testng.annotations.BeforeClass;
-
-public class BayesianPackageJson extends BayesianAbstractTestClass {
```

Comment:
is this removal che-starter related?

Reply:
This test is not used as far as issues with Bayesian are not planned to be fixed. I've added it here as a part of a cleanup. I'll mention it in a PR description.